p2p/http/libp2phttp.go

```diff
@@ -30,6 +30,7 @@ import (
 var log = logging.Logger("libp2phttp")
 
 const ProtocolIDForMultistreamSelect = "/http/1.1"
+const WellKnownProtocols = "/.well-known/libp2p/protocols"
```

Comment:
@MarcoPolo Changing this from `"/.well-known/libp2p"` to `"/.well-known/libp2p/protocols"` will break the ability to communicate with existing servers deployed without upgrading them first. Since these servers are running in places not under our control, the will no be immediately upgradable.

This means that libp2phttp needs to support the both the old and the new value, or revert to using the old value.

Suggest:
```go
const WellKnownProtocols = "/.well-known/libp2p"
```